aio/aio-proxy/aio_proxy/response/formatters/selected_fields.py

```diff
@@ -0,0 +1,27 @@
+from aio_proxy.labels.helpers import valid_fields_to_select
+
+
+def select_fields_to_include(include_fields, response: dict):
+    # Remove unselected fields from response
+    # In case `include_fields` is None, all extra fields are removed
+    if include_fields is None:
+        for field in valid_fields_to_select:
+            if field.lower() in response:
+                del response[field.lower()]
+        return response
+
+    for field in valid_fields_to_select:
+        if field not in include_fields and field.lower() in response:
+            del response[field.lower()]
+    return response
```

Comment:
the first `if include_fields is None:` is kinda weird. I would rather have :

- if minimal -> always remove all
- then add all fields in `include_fields`

But this would play much better if response was composed as we talked about during weekly. Maybe lets wait for a refacto

Reply:
I agree about the `None` condition. But as is, it will be complicated to implement this logic.